lib/screens/authentication/recover/recover_account_found/recover_account_found_screen.dart

```diff
@@ -74,7 +74,12 @@ class RecoverAccountFoundScreen extends StatelessWidget {
       case PageState.loading:
         return const FullPageLoadingIndicator();
       case PageState.failure:
-        return FullPageErrorIndicator(errorMessage: state.errorMessage);
```

Comment:
I did not change this IDK what this is from or where this came from?! huh.

Reply:
I guess some of this is due to merging in master. Hm.

Reply:
ohh